ext-apiserver/cmd/apiserver/main.go

```diff
@@ -32,5 +32,5 @@ import (
 
 func main() {
 	version := "v0"
-	server.StartApiServer("/registry/cluster.k8s.io", apis.GetAllApiBuilders(), openapi.GetOpenAPIDefinitions, "Api", version)
+	server.StartApiServer("/registry/k8s.io", apis.GetAllApiBuilders(), openapi.GetOpenAPIDefinitions, "Api", version)
```

Comment:
Is the `/registry` prefix okay? In some cases you want to share the etcd server and run your custom api-server with `--etcd-prefix="/cluster"`

Reply:
The `/registry` prefix is what was auto-generated. If the auto-generated code doesn't work with a shared etcd server and a custom apiserver flag then we should report a bug to sig-api-machinery to fix it in the code generation.

Reply:
@roberthbailey cool, thanks for the qualification